app/assets/javascripts/code/services/_draw_session.js

```diff
@@ -50,6 +50,7 @@ export const drawSession = (
 
       if (sensors.anySelected() && !sensors.tmpSelected() && session.last_hour_average) {
         level = calculateHeatLevel(heat, session.last_hour_average);
+        if (!level) { return  session.markers };
```

Comment:
You can remove `{}`. The nested if is a bit confusing maybe we can refactor around?